android/assets/jsons/translations/template.properties

```diff
@@ -291,6 +291,7 @@ Declare war on [nation] =
 Luxury resources = 
 Strategic resources = 
 Owned by you: [amountOwned] = 
+[cityName] ([population]) = 
```

Comment:
There is nothing to translate in this line. There is no point to add this to the templates.

Reply:
Hm... The idea was to handle RTL languages and maybe languages that would put the parentheses on the other side for other reasons (or not use parentheses maybe?)

Reply:
I'm in favor of simpler,  we can always complicate in the future if necessary